.github/workflows/recreate-qa-database.yml

```diff
@@ -0,0 +1,76 @@
+name: Replace Postgres Database
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Environment
+        default: qa
+        required: true
+
+env:
+ DOCKER_REPOSITORY: ghcr.io/dfe-digital/teaching-vacancies
+
+jobs:
+  turnstyle:
```

Comment:
Do we need a separate job? Turnstyle could be a step in the main job

Reply:
This could be changed similarly here: https://github.com/DFE-Digital/teaching-vacancies/blob/master/.github/workflows/deploy_app.yml#L20